plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -19,7 +19,8 @@ const languageDisplayNames = {
 	markdown: 'Markdown',
 	'atomics-filter': 'Atomics',
 	atomics: 'Atomics',
-	'abut-html': 'HTML'
+	'abut-html': 'HTML',
+	'html-no-indent': 'HTML'
```

Comment:
Totally okay if no, but is there a chance we could order the language name and modifier consistently? Maybe `{language}-{modifier}`, like `html-abut` and `html-no-indent`?

Reply:
You're definitely right @BenDMyers, I'd say for now why don't we consider updating `abut-html` in a follow up.

Reply:
Works for me!